buildkite/src/Jobs/Unit/Spec.dhall

```diff
@@ -0,0 +1,17 @@
+let S = ../../Lib/SelectFiles.dhall
+let D = S.PathPattern
+
+let JobSpec = ../../Pipeline/JobSpec.dhall
+
+in
+
+JobSpec::{
+  dirtyWhen = [
+    S.strictlyStart (S.contains "src/lib"),
+    S.strictlyStart (S.contains "src/nonconsensus"),
+    S.strictly (S.contains "Makefile"),
```

Comment:
`S.exactly` to only support a single *Makefile* or `S.contains` to support *Makefile**...